pennylane/capture/capture_qnode.py

```diff
@@ -87,6 +89,11 @@ def _(*args, qnode, shots, device, qnode_kwargs, qfunc_jaxpr, n_consts):
         mps = qfunc_jaxpr.outvars
         return _get_shapes_for(*mps, shots=shots, num_device_wires=len(device.wires))
 
+    def _qnode_jvp(*args_and_tangents, **impl_kwargs):
+        return jax.jvp(partial(qnode_prim.impl, **impl_kwargs), *args_and_tangents)
```

Comment:
The QNode primitive receives its own `JVP` rule here in order to be differentiable. It is simply a call to `jvp` acting on the `impl` call of the primitive. This dispatches to the standard derivative registered for QNodes in PennyLane with jax.

Reply:
Yes, because we're dispatching back to the standard computational path, which supports preprocessing. This is also why parameter-shift rule, and in fact any diff method working without capture, works with capture, too. It's just a wrapper at the moment.